webpack.config.js

```diff
@@ -4,7 +4,7 @@ var webpack = require('webpack')
 var HtmlWebpackPlugin = require('html-webpack-plugin')
 
 if (process.env.NODE_ENV === 'production') {
-    var publicPath = 'https://neontribe.github.io/ARCVMarket/';
+    var publicPath = './';
```

Comment:
@gingerbreadpup Why is this causing you issues with local dev? Your publicPath should only be overwritten by it if you are running in production env.